libr/core/cmd_type.c

```diff
@@ -267,8 +267,21 @@ static void save_parsed_type(RCore *core, const char *parsed) {
 	}
 }
 
+static Sdb *TDB_ = NULL; // HACK
+
 static int stdifstruct(void *user, const char *k, const char *v) {
-	return !strncmp (v, "struct", strlen ("struct") + 1);
+	r_return_val_if_fail (TDB_, false);
+	if (!strncmp (v, "struct", strlen ("struct") + 1)) {
```

Comment:
uhm, isnt this the same as !strcmp ? because you are comparing the trailing 0 too

Reply:
yes ... fixed old code c8c0efe31672fbed619a054654d368f12da668eb